cli/command/network/create_test.go

```diff
@@ -0,0 +1,182 @@
+package network
+
+import (
+	"bytes"
+	"io/ioutil"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/docker/cli/cli/internal/test"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/pkg/testutil"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/net/context"
+)
+
+func TestNetworkCreateErrors(t *testing.T) {
+	testCases := []struct {
+		args              []string
+		flags             map[string]string
+		networkCreateFunc func(ctx context.Context, name string, options types.NetworkCreate) (types.NetworkCreateResponse, error)
+		expectedError     string
+	}{
+		{
+			expectedError: "exactly 1 argument",
+		},
+		{
+			args: []string{"toto"},
+			networkCreateFunc: func(ctx context.Context, name string, createBody types.NetworkCreate) (types.NetworkCreateResponse, error) {
+				return types.NetworkCreateResponse{}, errors.Errorf("error creating network")
+			},
+			expectedError: "error creating network",
+		},
+		{
+			args: []string{"toto"},
+			flags: map[string]string{
+				"ip-range": "255.255.0.0/24",
+				"gateway":  "255.0.255.0/24",
+				"subnet":   "10.1.2.0.30.50",
+			},
+			expectedError: "Invalid subnet <nil> : invalid CIDR address: 10.1.2.0.30.50",
```

Comment:
Hmm yes. This is a bug! The code is using `s` instead of `subnet` in the error message. It looks like `net.ParseCIDR()` already includes the address, so the expected value should be  

```
invalid subnet: invalid CIDR address: 10.1.2.0.30.50
```

(invalid should start with a lowercase)

Could you update the code so to remove the `%s` that prints the `s` ?